ee/app/livechat-enterprise/client/views/app/customTemplates/livechatDepartmentCustomFieldsForm.html

```diff
@@ -2,25 +2,52 @@
 	<div class="input-line">
 		<label>{{_ "Max_number_of_chats_per_agent_description"}}</label>
 		<div>
-			<input type="number" class="rc-input__element customFormField" name="maxNumberSimultaneousChat" value="{{department.maxNumberSimultaneousChat}}" placeholder="{{_ "Max_number_of_chats_per_agent"}}" />
+			<input type="number" class="rc-input__element customFormField" name="maxNumberSimultaneousChat"
+				value="{{department.maxNumberSimultaneousChat}}" placeholder="{{_ "Max_number_of_chats_per_agent"}}" />
 		</div>
 	</div>
 	<div class="input-line">
 		<label>{{_ "How_long_to_wait_to_consider_visitor_abandonment"}}</label>
 		<div>
-			<input type="number" class="rc-input__element customFormField" name="visitorInactivityTimeoutInSeconds" value="{{department.visitorInactivityTimeoutInSeconds}}" placeholder="{{_ "Number_in_seconds"}}" />
+			<input type="number" class="rc-input__element customFormField" name="visitorInactivityTimeoutInSeconds"
+				value="{{department.visitorInactivityTimeoutInSeconds}}" placeholder="{{_ "Number_in_seconds"}}" />
 		</div>
 	</div>
 	<div class="input-line">
 		<label>{{_ "Livechat_abandoned_rooms_closed_custom_message"}}</label>
 		<div>
-			<input type="text" class="rc-input__element customFormField" name="abandonedRoomsCloseCustomMessage" value="{{department.abandonedRoomsCloseCustomMessage}}" placeholder="{{_ "Enter_a_custom_message"}}" />
+			<input type="text" class="rc-input__element customFormField" name="abandonedRoomsCloseCustomMessage"
+				value="{{department.abandonedRoomsCloseCustomMessage}}" placeholder="{{_ "Enter_a_custom_message"}}" />
 		</div>
 	</div>
 	<div class="input-line">
 		<label>{{_ "Waiting_queue_message"}}</label>
 		<div>
-      		<textarea class="rc-input__element customFormField" name="waitingQueueMessage" rows="4">{{department.waitingQueueMessage}}</textarea>
+			<textarea class="rc-input__element customFormField" name="waitingQueueMessage"
+				rows="4">{{department.waitingQueueMessage}}</textarea>
 		</div>
 	</div>
-</template>
+	<div class="input-line">
+		{{> livechatAutocompleteUser
+			onClickTag=onClickTagDepartment
+			list=selectedDepartments
+			onSelect=onSelectDepartments
+			collection='CachedDepartmentList'
+			endpoint='livechat/department.autocomplete'
+			field='name'
+			sort='name'
+			label="Department"
```

Comment:
Let's add another label to describe this feature more specifically. We can add something like:
`List of departments allowed for forwarding (Optional)`

WDYT?

We could, also, add an additional description below the input just like we do with the tag input:

![Screen Shot 2020-04-17 at 19 20 28](https://user-images.githubusercontent.com/59577424/79618575-cd15a980-80e0-11ea-89f6-f2577855a858.png)

The text could be something like:
`Allow to set a restricted list of departments that can receive chats from this department`;